app/jobs/dttp/register_for_trn_job.rb

```diff
@@ -5,9 +5,19 @@ class RegisterForTrnJob < ApplicationJob
     queue_as :default
     retry_on Client::HttpError
 
+    include ClockoverDependent
+
     def perform(trainee, created_by_dttp_id)
+      @trainee = trainee
+      @created_by_dttp_id = created_by_dttp_id
+
       return unless FeatureService.enabled?(:persist_to_dttp)
 
+      if before_clockover?
```

Comment:
Should we add a comment here to remove code after the clockover date?

Reply:
I've carded it up https://trello.com/c/QSG99PEx/1795-remove-clockover-guard-code